docker/docker-compose.yml

```diff
@@ -15,20 +15,20 @@ x-app-volume:
       target: /cuttle/src
 
 services:
-  database:
-    container_name: database
-    image: 'postgres:latest'
-    ports:
-      - 5432:5432
-    environment:
-      - POSTGRES_DB=cuttle
-      - POSTGRES_USER=cuttlesworth
-      - POSTGRES_PASSWORD=p4ssw0rd!
-      # Allow all connections, no password required (not recommended)
-      # - POSTGRES_HOST_AUTH_METHOD=trust
-    volumes:
-      - ./data/pgdata:/var/lib/postgresql/data
-    restart: always
+  # database:
+  #   container_name: database
+  #   image: 'postgres:latest'
+  #   ports:
+  #     - 5432:5432
+  #   environment:
+  #     - POSTGRES_DB=cuttle
+  #     - POSTGRES_USER=cuttlesworth
+  #     - POSTGRES_PASSWORD=p4ssw0rd!
+  #     # Allow all connections, no password required (not recommended)
+  #     # - POSTGRES_HOST_AUTH_METHOD=trust
+  #   volumes:
+  #     - ./data/pgdata:/var/lib/postgresql/data
+  #   restart: always
```

Comment:
I turned off the database service altogether b/c I haven't figured out how to make it dynamically configurable yet. It would be nice to be able to have different modes with and without postgres and with the server in different NODE_ENV's